tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -184,4 +185,15 @@ export class ScalarCardComponent<Downloader> {
       data: {cardId: this.cardId},
     });
   }
+
+  getLinkedTime(): LinkedTime {
+    return {
```

Comment:
nit: I understand there is a ngIf* to check if selected time is null.
but the linkedTime input of `scalar-card-linked-time-fob-controller` could still be null
I think it's better to add the check and then dropped the escalation later 
```
    if (selectedTime === null) {
      return null;
    }
```

Reply:
Done.